src/System.Management.Automation/engine/SessionState.cs

```diff
@@ -348,11 +348,11 @@ internal void InitializeFixedVariables()
 
             // $PSCulture
             v = new PSCultureVariable();
-            this.GlobalScope.SetVariable(v.Name, v, asValue: false, force: true, this, CommandOrigin.Internal, fastPath: true);
+            this.GlobalScope.SetVariableForce(v, this);
```

Comment:
Why was this changed?

Reply:
SetVariable() creates new variable by getting value as _constant_ from original variable but we need assign real object to always get fresh value.